### PR TITLE
[Trading][Exchange][WS] Add Binance, BinanceUS, Kucoin and CoinbasePro unauthenticated feeds

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
We should rename the .py tentacles files for them to be different from the name of their containing folder, otherwise it will create conflicts in python imports (avoid binance_websocket_feed/binance_websocket_feed.py, prefer binance_websocket_feed/binance_websocket.py or something like this)

File: Trading/Exchange/binance_websocket_feed/metadata.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "version": "1.2.0",
+  "origin_package": "OctoBot-Default-Tentacles",
+  "tentacles": ["BinanceWebsocketFeed"],
```
Comment (GuillaumeDSM) [inline Trading/Exchange/binance_websocket_feed/metadata.json:4]: should be the same as the name of the tentacle class (`BinanceCryptofeedWebsocketConnector`)

**Review by GuillaumeDSM — APPROVED** [review]
🎉 🌮 🐙

File: Trading/Exchange/binance_websocket_feed/binance_websocket.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import octobot_trading.exchanges as exchanges
+import cryptofeed.defines as cryptofeed_constants
+from octobot_trading.enums import WebsocketFeeds as Feeds
+
+
+class BinanceCryptofeedWebsocketConnector(exchanges.CryptofeedWebsocketConnector):
+    REQUIRED_ACTIVATED_TENTACLES = []
+    EXCHANGE_FEEDS = {
+        Feeds.TRADES: Feeds.UNSUPPORTED.value,
+        Feeds.KLINE: Feeds.UNSUPPORTED.value,
```
Comment (GuillaumeDSM) [inline Trading/Exchange/binance_websocket_feed/binance_websocket.py:25]:
```suggestion
        Feeds.KLINE: cryptofeed_constants.CANDLES,
```
with https://github.com/Drakkar-Software/OctoBot-Trading/pull/532 I propose we do it this way :)

Comment (Herklos) [reply Trading/Exchange/binance_websocket_feed/binance_websocket.py:25]: 👍

**Pull request merged**